### PR TITLE
refactor: add utility to format test results into array

### PR DESCRIPTION
**Pull request by maciej-ka**
Closes https://github.com/huozhi/sugar-high/issues/71
Hi, I'm sharing progress to get early feedback.
- [X] write helper
- [X] test it on one case
- [x] apply it to rest of cases
- [x] ignore break token (as it was before)
- [x] clean unused previous helpers

### CODE REVIEW

**Review by huozhi — COMMENTED** [review]
Thanks for the PR! I still want to reserve the getTokenValues utils so we can easily visualize the tokens change for a few one-line code places. Can you still keep that part in your change?

File: test/ast.test.js [hunk 1]
```diff
@@ -1,74 +1,59 @@
...
-      '/* evaluate */', '(', '19', ')', '/', '234', '+', '56', '/', '7', ';',
+    expect(extractTokenArray(tokens)).toEqual([
+      ["/* evaluate */", "comment"], ["(", "sign"], ["19", "class"], [")", "sign"], ["/", "sign"], ["234", "class"],
+      ["+", "sign"], ["56", "class"], ["/", "sign"], ["7", "class"], [";", "sign"]
     ])
   })
 
   it('calculation with defs', () => {
     const tokens = tokenize(`const _iu = (19) / 234 + 56 / 7;`)
-    expect(extractTokenValues(tokens)).toEqual([
```
Comment (huozhi) [inline test/ast.test.js:66]: Let's keep the extractTokenValues tests as well!

Comment (huozhi) [reply test/ast.test.js:66]: It's for visual review purpose, it's super easy to check if the tokenizer worked properly.

Comment (maciej-ka) [reply test/ast.test.js:66]: I agree. With added array wrap and type it's no longer instantly clear, how string is splitted into tokens.
Our ticket evolved into: "refactor type checks to include values".
I will add commit in a minute.

**Comment by maciej-ka** [comment]
Value checks returned. In rare cases where result has only one token I kept solo array check:

Screenshot 2023-09-21 at 23 38 07

**Review by huozhi — APPROVED** [review]
Thank you for making this refactor! ❤️

**Comment by maciej-ka** [comment]
Thanks also! I enjoyed mental flex part of it (code duplication - no way).

**Pull request merged**